src/locales/en.json

```diff
@@ -95,8 +96,8 @@
       "call_to_action": "Calculate approximate risk of contracting COVID",
       "delta_warning": "Risks have increased substantially, including for vaccinated people. <1>See blog post</1> for more details.",
       "delta_warning_heading": "Now updated for the Delta variant",
-      "omicron_warning": "Research is in progress to determine Omicron's impact on our models. The chance of infection is likely higher than the calculator currently shows, even if you are vaccinated.",
-      "omicron_warning_heading": "Not yet updated for the Omicron variant",
+      "omicron_warning": "<p>Vaccines without boosters are counted as much less effective against infection, but <em>unboosted vaccines still reduce your risk of severe or fatal illness.</em></p><p>Also not that COVID cases are reaching record highs in some areas, making activities riskier even if you are boosted.",
```

Comment:
typo? "Also not" should be "Also note" i think

Reply:
fixed, thanks!